### PR TITLE
Portfolio Works & About Section check

### PR DESCRIPTION
**Pull request by frankfembo**
- Add Portfolio and About sections with links to each projects repositories
- Setup buttons containing project links with hover upon interaction
- Add project sample images
- Add background images for each section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -109,6 +107,162 @@ <h5><strong>Skills:</strong></h5>
...
+                  <span class="vert-line"></span>
+                </ul>
+
+                <div class="abt-btns">
+                  <button type="button" class="butn btn">
+                    See Live  <a href="#" class="fa fa-power-off"></a>
+                  </button>
+                  <button type="button" class="butn btnl">
+                    See Source  <a href="#" class="fa fa-github"></a>
+                  </button>
```
Comment (cynthiainga) [inline index.html:259]: - [ ] In your design for the last project you have two buttons but in Figma design, there is only one for the `See project`. Also, change the color of the button :upside_down_face: :upside_down_face:
Your design
Screenshot from 2022-07-02 13-48-32

Figma design
Screenshot from 2022-07-02 13-49-41

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @frankfembo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors :heavy_check_mark: 
- [ ] Descriptive Pull Request :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 

## Required Changes ♻️

- [x] Please kindly check the comments under the review for more details :point_down: :point_down: .

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (frankfembo) [reply index.html:259]: Thanks a lot for the highlight, according to the figma template design that I chose (template 4) there are two buttons at the end of the About section ( [See Live] & [See Source] buttons), unlike the first template which was used as the example. Kindly, if you can check with figma template 4. Thank you once more

Comment (frankfembo) [reply index.html]: Ok my bad, sorry it was my mistake by the way, I instead included the mobile detail section

**Review by zhadier — APPROVED** [review]
## Hi @frankfembo  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
## Optional
- [ ] [_Optional_] Kindly add the button selector effects as specified in the Figma template below
image

<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by frankfembo** [comment]
PR approved and able to merge into main

**Pull request merged**